添加补充媒体位置权限(ACCESS_MEDIA_LOCATION)文档 (#153)
fluttercandies/flutter_wechat_assets_picker: README.md

`Glide` needs annotation to keep singleton, prevent conflict between instances and versions, so while the photo manager uses `Glide` to implement image features, the project which import this should define its own `AppGlideModule`. See [Android](#android) section for implementation.

> ### Remove Media Location permission

Reviewer: ```suggestion
### Disable `ACCESS_MEDIA_LOCATION` permission
```

Android contains `ACCESS_MEDIA_LOCATION` permission by default.

> This permission is introduced in Android Q. If your app doesn't need this permission, you need to add the following node to the Android manifest in your app.

Reviewer: ```suggestion
This permission is introduced in Android Q.
If your app doesn't need the permission,
you need to add the following node to the `AndroidManifest.xml` in your app:
```

`Glide` needs annotation to keep singleton, prevent conflict between instances and versions, so while the photo manager uses `Glide` to implement image features, the project which import this should define its own `AppGlideModule`. See [Android](#android) section for implementation.

> ### Disable `ACCESS_MEDIA_LOCATION` permission

Reviewer: ```suggestion
### Disable `ACCESS_MEDIA_LOCATION` permission

```
